docs/README.md

```diff
@@ -1,62 +1,102 @@
-* Anyone can register as an operator
-* Operators can begin providing services to an AVS
-* Stakers can delegate their stake to a single operator
-* Native ETH restaking is now fully featured, using beacon chain state proofs to validate withdrawal credentials, validator balances, and validator exits
-* Proofs are supported by beacon chain headers provided by an oracle (See [`EigenPodManager` docs](./core/EigenPodManager.md) for more info)
+#### Contents
+
+* [System Components](#system-components)
+* [Roles and Actors](#roles-and-actors)
 
 ### System Components
 
-**EigenPodManager**:
+#### EigenPodManager
 
-| File | Type | Proxy? | Goerli |
-| -------- | -------- | -------- | -------- |
-| [`EigenPodManager.sol`](../src/contracts/pods/EigenPodManager.sol) | Singleton | Transparent proxy | TODO |
-| [`EigenPod.sol`](../src/contracts/pods/EigenPod.sol) | Instanced, deployed per-user | Beacon proxy | TODO |
-| [`DelayedWithdrawalRouter.sol`](../src/contracts/pods/DelayedWithdrawalRouter.sol) | Singleton | Transparent proxy | TODO |
+| File | Type | Proxy |
+| -------- | -------- | -------- |
+| [`EigenPodManager.sol`](../src/contracts/pods/EigenPodManager.sol) | Singleton | Transparent proxy |
+| [`EigenPod.sol`](../src/contracts/pods/EigenPod.sol) | Instanced, deployed per-user | Beacon proxy |
+| [`DelayedWithdrawalRouter.sol`](../src/contracts/pods/DelayedWithdrawalRouter.sol) | Singleton | Transparent proxy |
 | [`succinctlabs/EigenLayerBeaconOracle.sol`](https://github.com/succinctlabs/telepathy-contracts/blob/main/external/integrations/eigenlayer/EigenLayerBeaconOracle.sol) | Singleton | UUPS proxy | [`0x40B1...9f2c`](https://goerli.etherscan.io/address/0x40B10ddD29a2cfF33DBC420AE5bbDa0649049f2c) |
 
 These contracts work together to enable native ETH restaking:
 * Users deploy `EigenPods` via the `EigenPodManager`, which contain beacon chain state proof logic used to verify a validator's withdrawal credentials, balance, and exit. An `EigenPod's` main role is to serve as the withdrawal address for one or more of a user's validators.
 * The `EigenPodManager` handles `EigenPod` creation and accounting+interactions between users with restaked native ETH and the `DelegationManager`.
-* The `DelayedWithdrawalRouter` imposes a 7-day delay on completing certain withdrawals from an `EigenPod`. This is primarily to add a stopgap against a hack being able to instantly withdraw funds (note that most withdrawals are processed via the `DelegationManager`, not the `DelayedWithdrawalRouter`).
+* The `DelayedWithdrawalRouter` imposes a 7-day delay on completing certain withdrawals from an `EigenPod`. This is primarily to add a stopgap against a hack being able to instantly withdraw funds (note users initiate withdrawals via the `DelegationManager`).
```

Comment:
I think this wording change may a step back?
maybe instead:
note that all withdrawals from EigenLayer -- other than those of "partial withdrawals" earned by validators -- are initiated via the `DelegationManager`

Reply:
good call

Reply:
made a slightly different change: [cda4c51](https://github.com/Layr-Labs/eigenlayer-contracts/pull/371/commits/cda4c5113f83693fd390a93313ed3b24960d78b0)

Reply:
it does omit the other 2 cases where the withdrawal router is being used (funds sent directly to pod and pre-"restaking activated" funds), but i think this captures the high level concept and the EPM docs can be more specific.